### PR TITLE
Add Credit Card Options

### PR DESCRIPTION
**Pull request by bangted**
Add Credit Card Options 
purposes are to validate the Credit Card Number, Expiry Date, and Card Security Code (CSC)

### CODE REVIEW

File: Sources/SuperValidator/Validators/CreditCard.swift [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        /// csc length should equal than card type
+        if csc.count != options.cardType.cardSecurityCodeLength {
+            return .failure(.invalidCSC)
+        }
+        return .success(())
+    }
+}
+
+extension String {
+    internal func toDate(withFormat format: String = "MM/yyyy") -> Date? {
```
Comment (andreyyoshua) [inline Sources/SuperValidator/Validators/CreditCard.swift:142]: I think this should be fileprivate or make the name more straightforward like `toCreditCardExpiryDate` maybe?

**Review by andreyyoshua — APPROVED** [review]
Overall looks good

**Review by alvinmatthewp — APPROVED** [review]

**Review by AmbarSeptian — APPROVED** [review]
bang please update the description PR ya

File: Sources/SuperValidator/Validators/CreditCard.swift (hunk 1 above)
Comment (bangted) [reply Sources/SuperValidator/Validators/CreditCard.swift:142]: Okay, I'll change to straight forward

**Pull request merged**